Sources/PeerConnectionManager+Peer.swift

```diff
@@ -90,7 +90,24 @@ public extension PeerConnectionManager {
     // - parameter toPeers: Specified `Peer` objects to send data.
 
     func sendData(_ data: Data, toPeers peers: [Peer] = []) {
-        session.sendData(data, toPeers: peers)
+        var peerRequested = peers.isEmpty == true ? allAvailablePeers : peers
```

Comment:
could you just add some comment to explain what's you did on this function ? Explain each step

Reply:
@Geoffrey-T for sure 👍, will ping you when it's ready

Reply:
@Geoffrey-T documentation available, any input welcomed 🎩